### PR TITLE
Fake Ops Bundle

### PR DESCRIPTION
**Pull request by yackemflam**
:cl:
* rscadd: Syndicate Nuke Ops Bundle has been added to throw the crew off a little

### CODE REVIEW

**Comment by calzilla1** [comment]
This seems a bit more extreme compared to the other bundles

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 1]
```diff
@@ -108,6 +108,19 @@
...
+				new /obj/item/clothing/head/helmet/space/syndicate(src)
+				new /obj/item/clothing/suit/space/syndicate(src)
+				new /obj/item/weapon/gun/projectile/automatic/c20r(src)
+				new /obj/item/ammo_storage/magazine/a12mm/ops(src)
+				new /obj/item/weapon/card/id/syndicate(src)
+				new /obj/item/weapon/implanter/explosive(src)
+				new /obj/item/clothing/shoes/magboots/syndie(src)
+				new /obj/item/clothing/under/syndicate(src)
+				new /obj/item/clothing/mask/gas/syndicate(src)
+				new /obj/item/clothing/gloves/combat(src)
```
Comment (Kurfursten) [inline code/game/objects/items/weapons/storage/uplink_kits.dm:121]: Did you test this? Not totally sure more than 7 items will work

**Review by PJB3005 — APPROVED** [review]

**Comment by Probe1** [comment]
What @calzilla1 said.

File: code/game/objects/items/weapons/storage/uplink_kits.dm (hunk 1 above)
Comment (yackemflam) [reply code/game/objects/items/weapons/storage/uplink_kits.dm:121]: The balloon bundle gives you more than 7 balloons and it works

Comment (AutismSpider) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: Two bundles that give more then 7 items the balloons and the axe killer kit.

**Comment by SonixApache** [comment]
>C-20r outside of ops
>syndiboots outside of ops
>syndiemasks outside of ops

fuck off

**Comment by Blithering** [comment]
Why would you want to fake ops? Ops being sighted is an instant shuttle call and call to arm security. At best you'll get a few kills before getting mobbed to death and then everyone goes home on the shuttle.
Even if you manage to get your target during that time and had the die a glorious death objective, you've kinda wasted the entire round for everyone else.

**Comment by calzilla1** [comment]
Though, it does seem that that the other bundles seem due for a buff

**Comment by ghost** [comment]
@Blithering that's the point nerd

File: code/game/objects/items/weapons/storage/uplink_kits.dm (hunk 1 above)
Comment (ghost) [reply code/game/objects/items/weapons/storage/uplink_kits.dm:121]: The miami gear has 10 objects

**Comment by BobdaBiscuit** [comment]
A decent bundle for guaranteed hilarity.

File: code/game/objects/items/weapons/storage/uplink_kits.dm (hunk 1 above)
Comment (Exxion) [reply code/game/objects/items/weapons/storage/uplink_kits.dm:121]: You just won't be able to put it all back in after you take it out.
Not that you could anyway, considering some of the stuff has a higher `w_class` than a box can hold.

**Comment by strawberrytan** [comment]
I approve of keeping the crew on their toes

**Comment by troots** [comment]
just powersink roundstart to get powergamers to swarm to the nuke disc

**Comment by Frido-lin** [comment]
Add a paper with the nuke code

**Comment by yclatious** [comment]
>giving a highly advanced syndicate nuclear operative strike force suit to a random traitor
>giving a fucking cr20, a gun with good ammo and a stun on hit, to a normal traitor
>giving a explosive implant and a agent card to boot

Like blithering said, what this accomplishes is making sure that the round in less than 20 minutes, its attempting to be anti meta while not realizing itl just make rounds end faster because some chucklefuck went with a bundle and murderboned, not to mention that giving some random untrained traitor a high value operative suit, a extremely good gun,  a extra mag of ammo, a agent card and a explosive implant makes both no sense in MUH LORE and is ridiculous value for a bundle, wouldn't you think so?

**Comment by Probe1** [comment]
It isn't a nuke op hardsuit.  It's a softsuit.  It does come with the magboots though.

**Comment by yclatious** [comment]
Huh, apologies then, though the intention was to make people call it over seeing the actual nuke suits, still think its gives you a ton of goodies and that its bordering on overkill, a cr20 is a godlike gun, the explosive implant is great, the agent card aswell and its still a rather robust piece of space worthy armor, wouldn't you say? Id even say its powercreep compared to every other bundle.

**Comment by eplgr** [comment]
I really like the idea of confusing people. Not really knowing what roundtype you are playing is cool.
But yeah, it does seem way better than any other bundle.,

**Comment by SonixApache** [comment]
how about you just replace everything in this bundle with a toy version
toy gun
toy suit
toy boots
fake ID

**Comment by Probe1** [comment]
Just tone down how much they get.  Right now you have about 2-3 bundles worth of stuff in one.

```

	new /obj/item/clothing/head/helmet/space/syndicate(src)
		new /obj/item/clothing/suit/space/syndicate(src)
		new /obj/item/clothing/shoes/magboots/syndie(src)
		new /obj/item/clothing/under/syndicate(src)
		new /obj/item/clothing/mask/gas/syndicate(src)
		new /obj/item/clothing/gloves/combat(src)

```
This is in itself a bundle.

```

		new /obj/item/weapon/gun/projectile/automatic/c20r(src)
		new /obj/item/ammo_storage/magazine/a12mm/ops(src)

```
ditto
```

		new /obj/item/weapon/card/id/syndicate(src)
		new /obj/item/weapon/implanter/explosive(src)

```
ditto

**Comment by Exxion** [comment]
How is an agent ID and an E-implant a whole bundle?
How is one gun and ammo for it a whole bundle?
Remember that a bundle must have AT LEAST 10 TC worth of stuff in it.

**Comment by yackemflam** [comment]
20 tc worth of stuff at least, not 10 tc. we doubled tc a long while ago

**Comment by SonixApache** [comment]
do we really need more ezmode round enders

**Comment by calzilla1** [comment]
So, a guy rolls up, pops an assistant, gets lynched and it's OMG ROUND DESTROYER , jesus you guys really don't know what a REAL round killer is

**Comment by yackemflam** [comment]
Tbh, I dont truely know what would happen if this gets merged
But I don't think this would be a complete round ender.

**Comment by Probe1** [comment]
@Exxion you're comparing incomparable things.  Bundles give items not normally accessible to traitors.  I was not saying that there SHOULD be 3 bundles added. I was saying there were 3 bundles worth of items.  

@yackemflam Just work on balancing it by slimming the list instead of 'a one man nuke op'.

**Comment by Anton-Kr** [comment]
Leave it as it as
Half the probability
Problem solved.

**Comment by Blithering** [comment]
It's not a round killer because it's too powerful, it's a round killer because the moment someone spots you, the shuttle is getting called and not getting recalled.

**Comment by Exxion** [comment]
@Probe1 I didn't say you said that three bundles should be added. I said that it was not three bundles worth of items.
The third split you suggested are both items that traitors CAN normally get, and for much less than 20 TC

**Comment by ComicIronic** [comment]
An unfun bundle does not become better just because it's less likely to appear.

**Comment by Probe1** [comment]
What Comic said.

@Exxion Fair enough.  I'm not trying to find a way to balance it by division.  I was drawing an example of how the bundle is three different things in one.

**Comment by yclatious** [comment]
A cr20 and a mag itself is worth atleast 15 tcs, its a gun with good damage and stun alongside taking other mags of non cr20 ammo, explosive implant is 12 tc if the wiki isn't outdated, sydie space suit is 4 if the wiki isnt outdated, agent is 2 itwio, meaning its atleast worth 35 tc, this bundle is just too good compared to anything else.

The though is nice and the concept is interesting but if someone sees a cr20 nobody is going to stick around and the shuttle isn't getting recalled. Its a round killer, who wants to actually try and see if its real ops or not, the shuttle is getting called, the screen shot/the board stolen and the crew will either go full validhunter mode or let the nukeops trample over everyone, but nobody is recalling the shuttle.

Don't give near admintier guns/special mode/special team guns to normal traitors, even if from a bundle, especially when they can make a round end by player behaviour. You can make it rare to get but it doesn't make it any better, it just makes the bad appear less instead of making the bad disapear like it should.

**Comment by SonixApache** [comment]
@calzilla1 I've succesfully had the shuttle called as traitor AI just by pretending I saw a nuke op. 
They're a round ender.

**Comment by unusualcrow** [comment]
This isn't at all worth 10 TC. Nerf the bundle or increase it to 20 TC.

**Comment by JustSumBody** [comment]
Reminder that the screwed bundle exists and is still 10 times worse than this bundle ever will be.

**Comment by yclatious** [comment]
>we already have bad content
>lets get more bad content based on the fact we already have bad content

Is this quite literally not a case of slippery slope? Because it sure sounds like one to me.

**Comment by AutismSpider** [comment]
@JustSumBody nah man people love getting shit on by two max cap bombs or that one that gives you all the tools to set the singo loose.

**Comment by Probe1** [comment]
Something being shit has never been and will never be justification for adding more shit.  "Should we clean up this mess?  No lets make it worse".

Anyhow @yackemflam you've gotten pretty clear feedback on how to get this PR into a merge state.  You'll have to follow that if you want it merged.

**Comment by BobdaBiscuit** [comment]
Sneaky c*llabs do it again!

**Comment by ghost** [comment]
After seeing what a guy in a nuke ops hardsuit can do on lowpop, no thank you.

Maybe have it have a minimum population lock?

**Comment by yclatious** [comment]
You can lower the chances of cancer all you want but you shouldn't have cancer to begin with yacke. Bad bundle with too much value, cr20 alone is near admin tier good, you can make it as rare as you want to but it shouldn't be available to a traitor anyways, even if you are restricting it by fucking luck.

Its a explosive implant, a cr 20, a extra mag of ammo, a robust sydnie softsuit, magboots and a gas mask, all in a bundle that is almost sure to make people recall on high pop because they don't want to die to ops and will be the best murderboning tool possible in low pop due to giving away a near admin tier gun with good ammo, good damage and stun on hit, id be hardpressed to see someone in low pop who doesn't recall when a traitor has a damaging balistic taser and softsuit with nuke ops tier protection.

Its cancerous garbage.

**Comment by Probe1** [comment]
Lowering the chance was not the feedback I was referring to for this to be in a merge state.   

https://github.com/vgstation-coders/vgstation13/pull/14400#issuecomment-288121455
https://github.com/vgstation-coders/vgstation13/pull/14400#issuecomment-287899272
https://github.com/vgstation-coders/vgstation13/pull/14400#issuecomment-287826030
https://github.com/vgstation-coders/vgstation13/pull/14400#issuecomment-288169165
https://github.com/vgstation-coders/vgstation13/pull/14400#issuecomment-288167709

**Comment by Yelier** [comment]
Perhaps a red spacesuit, tacticool turtleneck, jackboots, SWAT mask, oxygen jetpack, and RPG launcher instead? 

Red hardsuit is really fucking strong, and so is the cr20. A nice bundle is one or two traitor items, maybe an exclusive item, some flavor items, and stuff to compliment the other item. 

A normal red spacesuit is both an option (albeit not a good one) for ops and won't get the shuttle called, the tacticool gear is just for flavor, the RPG launcher (maybe it'd have 1 or 2 extra shots?) would be the special item, and the jetpack is a useful tool that compliments the rest.

**Comment by AutismSpider** [comment]
@Probe1 what makes you think any of that makes good bundle need pull your head out of your ass m8

**Comment by AutismSpider** [comment]
@Yelier red spacesuit, tacticool turtleneck, jackboots, SWAT mask, oxygen jetpack 357 revolver and speed loader rename it to ops jr or in training bundle.

**Comment by eplgr** [comment]
If the idea is to fool the crew, you only need:
- Red space suit
- cr20

With cr20 being the key to do it. I don't think that'd be imbalanced. No magboots, no syndie mask, no extra ammo, no gloves, no syndie agent ID.
It'd still be an useful bundle and not too overpowered, in my opinion. In fact we could discuss whether adding an explosive implant on top of that would cross the line or it wouldn't.

**Comment by Anton-Kr** [comment]
Ops get fucking smashed all the time, especially when alone. I don't see what everyone is bitching about balance. It's a bundle that will appear practically never.

**Comment by AutismSpider** [comment]
@IsdatAfamas most people hit the bundle button for the max cap bomb one others are down right useless or big old meme.

**Comment by yackemflam** [comment]
Because coders threaten to close my bundle if things dont go their way 100%

**Comment by Anton-Kr** [comment]
Thts the whole point of it being a random bundle. By some logic in this thread a five member nuke ops team is able to kill everyone with ease with their super op Cr20s.

**Comment by yclatious** [comment]
You act like the main cause of ops failing is anything but borgs, the rarity of the role to people, ops rolling newbies more than veterans who actually know how to ops and people who want to murderbone instead of planing the disk extraction or often ebough, a combination of all these.

The issue about ops sucess is the players cant use the tools and coordinate properly enough to use them to the fullest extent, because ops can, and if coordinated enough, will kill everyone with the stock cr20s alone, assuming no borgs decide to try and fuck everything up and that nobody brought a ion.

Making a busted fucking strong gun, a explosive implant, a robust suit and so on acessible but rare to the average traitor doesnt make it any less busted when you are giving the non admin version of the best balistic gun out there, a fucking balistic weapon that stuns when we go by stun and done combat with tons of ammo and even a extra mag to boot, unless I misread the file.

**Comment by ghost** [comment]
If you're gonna be insistant on it being nuke op lite, try the micro smg in place of the CR.

Makes sense considering the size, not sure on the rest of the stats however

**Comment by SonixApache** [comment]
>rig/syndi
>syndie hardsuits outside of nuke ops

please stop
it's bad enough when someone finds one in a vault

**Comment by yclatious** [comment]
So we went from gun and massive value with a decent chance of ops to "red suit, fucking call it" armory raiding.

Quit fucking taking shit from unique gamemodes and allowing them to be in normal gameplay without bus or hard as fuck vault diving, especially shit like this that actually makes people recall the fucking shuttle because, lets be real here, who actually wants to try and test if its fucking ops who are here to get that fukken disk? Who here would do that? Who would, in a position of power to be able to call and recall the emergency shuttle, go ahead and risk a bunch of murderboning faggots to nuke the station unless they metad that the pop wasn't enough to actually get them?

**Comment by SonixApache** [comment]
should add an antag bundle that gives you a stone mask, blob seed, xenomorph juice and changeling vial

**Comment by Aurx** [comment]
Instead of not giving them a full op kit, why not just give them only part of a mag and a lookalike hardsuit without the stat values?  I'd think that would solve the balance issues better while still letting them have the OH GOD OPS effect.

**Comment by Exxion** [comment]
I think the "OH GOD OPS" effect is what a bunch of the complaints here are about
But yeah, full ops hardsuit protection values are pretty nuts

**Comment by invoid9** [comment]
I don't think there are many antag items/abilities/spells that are there to fool the crew into thinking its another roundtype. But, a well done example of this is the AI fake blob message. It adds no ability/items/powerups to the AI whatsoever, its spurs the crew into some sort of action (which may or may not benefit the AI), it's fairly convincing for a time, it adds a whole new flavor to the round without actually stepping on the toes of a blob round too heavily, and it ultimately matters to the crew whether it was fake or not. 

Compare that to this bundle that gives a traitor a full set of some of the best equipment in the game,  means a shuttle call immediately once seen, won't be really convincing, will most likely turn the round into another murderbone fest, cheapens a lot of the items and feeling unique to ops,  and ultimately does not matter to the crew whether it is actually ops. The shuttle will be called, a ton of people will probably die. I think if there's gonna be more abilities/items that do this kind of thing, it should be done really well, otherwise it just looks like an excuse to bring items you like from one roundtype into another. (inb4 there's another example of this sort of thing that does it badly that I overlooked or didn't think of. emagged fake tome is on a much smaller scale, and requires actual effort on the part of the traitor to pull off)

And I don't think the random element is any excuse to add this anyway.

**Comment by yackemflam** [comment]
Ops are very weak against lasers in the first place. It's only robust against melee and bullets. Which are less used from NT employees.

**Comment by SonixApache** [comment]
Please remember that nuke op hardsuits are basically equal in protection values to generic syndie softsuits.

The issue is that this is an ezmodo round ender, or it'll fuck up the way ops rounds go. Either way, it's a bad situation for the crew.

**Comment by Fruktlimpa** [comment]
As everyone has pointed out the issue is the round-ending effect rather than the balance effect.

**Comment by Arthurdentist** [comment]
this bundle devalues the grandprize bundle.

**Comment by Hoshi-chan** [comment]
Rename it to "Instant Shuttle Call", and instead of actually providing nuke ops gear, provide five inflatable nuke op decoys and a chameleon projector.

**Comment by Probe1** [comment]
Well, we tried to work with you on this.

**Pull request closed without merging**